### PR TITLE
Add wrapper for CMS support to the Swift package

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Use the existing code to implement a wrapper for providing CMS encryption to this package. This will create the XCFramework binary that can be used in Swift packages.

### CODE REVIEW

File: output/CMSSupport.xcframework/ios-arm64/CMSSupport.framework/Headers/CMSSupport.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
+//
+//  CMSSupport.h
+//  CMSSupport
+//
+//  Created by Erin Mounts on 6/12/15.
+//
+//	Copyright (c) 2105, Sage Bionetworks
```
Comment (Erin-Mounts) [inline output/CMSSupport.xcframework/ios-arm64/CMSSupport.framework/Headers/CMSSupport.h]:
```suggestion
//	Copyright (c) 2015, Sage Bionetworks
```
oops

File: output/CMSSupport.xcframework/ios-arm64_x86_64-simulator/CMSSupport.framework/Headers/CMSSupport.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
+//
+//  CMSSupport.h
+//  CMSSupport
+//
+//  Created by Erin Mounts on 6/12/15.
+//
+//	Copyright (c) 2105, Sage Bionetworks
```
Comment (Erin-Mounts) [inline output/CMSSupport.xcframework/ios-arm64_x86_64-simulator/CMSSupport.framework/Headers/CMSSupport.h]:
```suggestion
//	Copyright (c) 2015, Sage Bionetworks
```

File: CMSSupport/CMSSupport.h [hunk 3]
```diff
@@ -30,7 +30,7 @@
 //	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (Erin-Mounts) [inline CMSSupport/CMSSupport.h:30]: Please fix my 6-year-old typo in the copyright year up above here, thx

**Review by Erin-Mounts — APPROVED** [review]
Couple of nits for six-years-ago me, otherwise looks good

File: CMSSupport/CMSSupport.h (hunk 3 above)
Comment (syoung-smallwisdom) [reply CMSSupport/CMSSupport.h:30]: whoops, missed that one.

**Pull request merged**